src/providers/fail_over.h

```diff
@@ -81,6 +81,7 @@ struct fo_options {
     time_t srv_retry_neg_timeout;
     time_t retry_timeout;
     int service_resolv_timeout;
+    bool dns_search;
```

Comment:
The other fields are explained in the comment above. Should this one be documented?

Reply:
Good point, updated.